Tsickle distribution package contains multiple amd-module declaratios.

The v0.30.0 distribution of Tsickle is invalid. some files contain `amd-module` declarations more than once. Having multiple declarations of `amd-module` is invalid TypeScript file.

Example: `node_modules/tsickle/src/googmodule.d.ts`
```
/// <amd-module name="tsickle/src/googmodule" />
/**
 * @license
 * Copyright Google Inc. All Rights Reserved.
 *
 * Use of this source code is governed by an MIT-style license that can be
 * found in the LICENSE file at https://angular.io/license
 */
/// <amd-module name="tsickle/src/googmodule" />
import {ModulesManifest} from './modules_manifest';
import * as ts from './typescript';

```

Here is a full list of offending files:

```
node_modules/tsickle/src/googmodule.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/googmodule" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/modules_manifest.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/modules_manifest" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/source_map_utils.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/source_map_utils" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/tsickle.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/tsickle" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


node_modules/tsickle/src/typescript.d.ts:9:1 - error TS2458: An AMD module cannot have multiple name assignments.

9 /// <amd-module name="tsickle/src/typescript" />
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -35,9 +35,9 @@ http_archive(
 
 http_archive(
     name = "build_bazel_rules_typescript",
-    url = "https://github.com/bazelbuild/rules_typescript/archive/0.15.0.zip",
-    strip_prefix = "rules_typescript-0.15.0",
-    sha256 = "1aa75917330b820cb239b3c10a936a10f0a46fe215063d4492dd76dc6e1616f4",
+    url = "https://github.com/rkirov/rules_typescript/archive/v0.16.0.zip",
+    strip_prefix = "rules_typescript-0.16.0",
+    sha256 = "f5aedd3a792e5af19cd0c0f0318cb692e2989e816e896e794152d07808fccacd",
 )
 
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_setup_workspace")

